pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -1211,6 +1211,23 @@ let
         };
       };
 
+      gencay.vscode-chatgpt = buildVscodeMarketplaceExtension {
+        meta = {
+          changelog = "https://marketplace.visualstudio.com/items/gencay.vscode-chatgpt/changelog";
+          description = "ChatGPT, GPT-3 and Codex conversations in Visual Studio Code";
```

Comment:
```suggestion
          description = "A Visual Studio Code extension to support ChatGPT, GPT-3 and Codex conversations";
```

Reply:
I'm sorry for the late suggestion. It was pending at GitHub interface. It is a thing at GitHub that is counter-intuitive that sometimes holds back review. I didn't see it was pending.

Reply:
NP!

About your suggestion, I prefer to leave it as it is, I copy pasted the description from the homepage of the project.

Reply:
That is not possible because it has to say it is a VSCode extension. Notice, nixpkgs is a general repository and not exclusive to VSCode extensions.

Reply:
And there is a 'right' (as in expected) way to word the description. Care to look at others and you might see the pattern.

Reply:
Also you don't need to make a change. Because I'll squash it. I have commited it myself.

Reply:
Fair enough, I'll pay attention next time. Thanks !

Reply:
Could you please keep the authorship on the commit ?

Reply:
Which part you mean specifically?

Reply:
Please ignore me, wrong thread !

Reply:
Good. I'm waiting CI to have this merged.